pages/design/rule-design.md

```diff
@@ -2,35 +2,69 @@
 title: Rule Design
 ---
 
-The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is suggested:
+The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is based on [ACT Rules Format 1.0](https://www.w3.org/TR/act-rules-format/):
```

Comment:
WCAG-ACT-RULES-CG can be hyperlinked?

Reply:
Should it be? It isn't in the current document.

Reply:
I don't believe a change is needed here, so resolving this comment